DragaliaAPI/AutoMapper/Profiles/EmblemReverseMapProfile.cs

```diff
@@ -0,0 +1,22 @@
+﻿using AutoMapper;
+using DragaliaAPI.Database.Entities;
+using DragaliaAPI.Models.Generated;
+
+namespace DragaliaAPI.AutoMapper.Profiles;
+
+public class EmblemReverseMapProfile : Profile
+{
+    public EmblemReverseMapProfile()
+    {
+        this.AddGlobalIgnore("DeviceAccount");
+        this.AddGlobalIgnore("Owner");
+
+        this.CreateMap<EmblemList, DbEmblem>()
+            .ForMember(x => x.EmblemId, o => o.MapFrom(x => x.emblem_id))
+            .ForMember(x => x.GetTime, o => o.MapFrom(src => src.gettime))
+            .ForMember(x => x.IsNew, o => o.MapFrom(src => src.is_new));
```

Comment:
for emblemId and isNew the naming conventions should be able to figure it out -- or are you just preferring to be explicit

Reply:
I'll remove it